packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/data_source/pin_biometrics_local_data_source.dart

```diff
@@ -0,0 +1,58 @@
+// {{> licence.dart }}
+
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import '../../base/data_sources/local/shared_preferences_instance.dart';
+
+/// You have to implement and provide a [BiometricsLocalDataSource], you can
+/// store the value of [_areBiometricsEnabled], for example in [SharedPreferences]
+class PinBiometricsLocalDataSource implements BiometricsLocalDataSource {
+  PinBiometricsLocalDataSource(
+    this._sharedPreferences,
+  );
+
+  bool? onRestart;
+  final SharedPreferencesInstance _sharedPreferences;
+
+  /// This value is the state for the user general choice for the app
+  static const _areBiometricsEnabled = 'areBiometricsEnabled';
+
+  /// This value is changed in while using the app to stop and start auto biometrics
+  static const _areBiometricsEnabledWhileUsingTheApp =
+      'areBiometricsEnabledWhileUsingTheApp';
+
+  Future<void> temporaryDisableBiometrics(bool disable) async =>
+      await _setBoolValue(_areBiometricsEnabledWhileUsingTheApp, !disable);
+
+  @override
+  Future<bool> areBiometricsEnabled() async {
+    if (onRestart == null) {
+      var areBiometricsEnabled = await _getBoolValue(_areBiometricsEnabled);
```

Comment:
Here you are still using var even tho the value of `areBiometricsEnabled` can be final